File: EQIS_env-main/react_sample/react/src/interfaces/lang/langListSearch.ts
import { cloneDeep } from 'lodash';

import { Ibase } from '@/interfaces/common/base';

export interface IlangListSearch extends Ibase {
    mlgCd: string;
    langCd: string;
    mlgSbc: string;
}

export const langListSearchInit = () => {
    const langListSearch: IlangListSearch = {
        mlgCd: '',
        langCd: '',
        mlgSbc: '',
    };

    return cloneDeep(langListSearch);
};

File: EQIS_env-main/react_sample/springboot/Spring/src/main/java/com/sample/basic/sample/code/model/LangListSearch.java
package com.sample.basic.sample.code.model;

import com.sample.basic.cmm.model.Base;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

@Data
@EqualsAndHashCode(callSuper=false)
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class LangListSearch extends Base{
	private String mlgCd;
	private String langCd;
	private String mlgSbc;
}
